setting-up-appsmith/aws-ami.md

```diff
@@ -0,0 +1,185 @@
+# Introduction
+This tutorial will walk you through the process of using the AWS Console to create and provision a new AWS cloud server. Since AWS offers a Free Tier valid for 12 months, you will be able to give Appsmith a whirl without worrying about being billed for usage.
```

Comment:
Instead of -
`This tutorial will walk you through the process of using the AWS Console to create and provision a new AWS cloud server.`
Let's say -
`This section will walk you through the steps of setting up Appsmith on AWS.

Reply:
Is there a reason that we want to mention `Since AWS offers a Free Tier valid for 12 months, you will be able to give Appsmith a whirl without worrying about being billed for usage` ?

Reply:
No real reason. Just wanted to emphasize that the user will not be billed. Have removed this sentence because it doesn't add a lot of value.